webtau-http/src/main/java/com/twosigma/webtau/http/HttpResponse.java

```diff
@@ -70,7 +70,7 @@ public boolean hasContent() {
     }
 
     public boolean isJson() {
-        return contentType.contains("/json");
+        return contentType.matches(".*/.*\\+?json");
```

Comment:
wonder if adding regepx now warrants a unit test to test negative cases

Reply:
What sort of negative test did you have in mind?  Something like responding with `application/notreallyjson`?

I was in two minds whether to do this via regex to be honest.  I wondered about `endsWith(“json”)` but that would also accept things like `notreallyjson` which I don’t think is correct.  Also considered `contains(“/json”) || contains (“+json”)`.  I chose regex because it felt more correct but I could probably be convinced otherwise.

Reply:
Yes, something like the example you have.

Reply:
added